src/convert.jl

```diff
@@ -1132,3 +1132,7 @@ function convert(::Type{Zonotope}, cpa::CartesianProductArray{N, AZ}) where
     G = reduce(blockdiag, sparse.(genmat.(arr)))
     return Zonotope(c, G)
 end
+
+convert(::Type{HPolytope}, P::HPolyhedron) = HPolytope(copy(constraints_list(P)))
```

Comment:
i'm surprised that these functions copy; i didn't expect it to be the case, see for example https://github.com/JuliaReach/LazySets.jl/blob/master/src/convert.jl#L177

Reply:
Just to be clear, I didn't change things here, just moved the code.